src/Models/UsersModel.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+
+require_once 'src/Entities/User.php';
+
+
+class UsersModel
+{
+
+    private PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    public function getAllUsers()
+    {
+
+        $query = $this->db->prepare('SELECT `id`, `user-name`, `email-address`, `password` FROM `users`;');
+        $query->execute();
+        $data = $query->fetchAll();
+
+        return $this->hydrateMultipleUsers($data);
+    }
+
+    public function selectUser(string $inputtedUsername): User|null
```

Comment:
The story asks for login to be done based on email rather than username - I would prefer to stick with that even though my front end code has username.

How much work would it be to switch?